### PR TITLE
remove fake tabs, use navs

### PR DESCRIPTION
**Pull request by amazingphilippe**
# Summary | Résumé

This updates our pill component to feel just like a nav to screen reader users. The pill component already worked like a nav, so the markup doesn't really changes. I did simplify the component a bit, especially for the label and did some CSS tweaks that should be unoticeable. 

(Fixes) https://github.com/cds-snc/notification-planning/issues/498

# Test instructions | Instructions pour tester la modification

These are the endpoints you can use to view the different pill component instances

- /monthly
- /notifications/email?status=delivered
- /templates/email

1. Check that the nav is labelled with aria-label, in both languages
2. Check that the links work, and land you where you expect

A11y QA:
- [x] Scan with ARC toolkit or Axe devtools. For issues related to the pill
- [x] Test the pill with voiceover on Safari
- [ ] Test the landmark menu with voiceover on Safari

### CODE REVIEW

File: app/assets/stylesheets/tailwind/components/pill.css [hunk 1]
```diff
@@ -81,7 +83,7 @@
   .pill-separate-item:hover,
   .pill-separate-item:focus,
   .pill-separate-item:link:focus {
-    @apply text-blue-lightblue25;
+    @apply bg-yellow text-blue;
```
Comment (smcmurtry) [inline app/assets/stylesheets/tailwind/components/pill.css:86]: This is causing somewhat weird behaviour where only the background of the selected link of the pill turns blue when you hover, so now the hover styles are inconsistent. Before, all links in the pill turned blue background on hover. The links already have an underline that appears on hover, so maybe we can just remove the background colour hover style?

Here's a demo - 1st our current app then your review app: https://youtu.be/XK3egR4w1Vw

File: app/templates/components/pill.html [hunk 2]
```diff
@@ -4,38 +4,27 @@
   items=[],
   current_value=None,
   big_number_args={'smaller': True},
-  show_count=True
+  show_count=True,
+  label=""
 ) %}
-  <div role='tablist' class='pill flex'>
+  <nav class='pill flex' aria-label="{{ label }}">
```
Comment (smcmurtry) [inline app/templates/components/pill.html:10]: Nice!

**Review by smcmurtry — CHANGES_REQUESTED** [review]
This is great @amazingphilippe! I like the way the `<nav>` is working, it seems much clearer using voiceover:

Current prod: https://youtu.be/-rie7OE7ne4
Your review app: https://youtu.be/nygf3HhpcLY

The only thing I noticed is an issue with the focus styles - can you take a look at that before merging?

File: app/templates/components/pill.html [hunk 3]
```diff
@@ -4,38 +4,27 @@
   items=[],
   current_value=None,
   big_number_args={'smaller': True},
-  show_count=True
+  show_count=True,
+  label=""
```
Comment (jimleroyer) [inline app/templates/components/pill.html:8]: Interesting, looks simpler now in the implementation. 👍

File: app/assets/stylesheets/tailwind/components/pill.css (hunk 1 above)
Comment (amazingphilippe) [reply app/assets/stylesheets/tailwind/components/pill.css:86]: Oh yeah, that is weird. I wanted to simplify this CSS a bit. Good catch!

Comment (amazingphilippe) [reply app/assets/stylesheets/tailwind/components/pill.css:90]: Should be fixed! I reviewed CSS specificity, and found out about LVHA ordering. added a comment for future reference.

**Review by smcmurtry — APPROVED** [review]
Looks good to me!

**Pull request merged**